lib/indexer.rb

```diff
@@ -0,0 +1,63 @@
+module Argo
+  class Indexer
+    ##
+    # Returns a Logger instance for recording info about indexing attempts
+    # @yield attempt to execute 'entry_id_block' and use the result as an extra identifier for the log 
+    #   entry.  a placeholder will be used otherwise. 'request.uuid' might be useful in a Rails app.
+    def self.generate_index_logger(&entry_id_block)
+      index_logger = Logger.new(Settings.INDEXER.LOG, Settings.INDEXER.LOG_ROTATION_INTERVAL)
+      index_logger.formatter = proc do |severity, datetime, progname, msg|
```

Comment:
Unused block argument - `severity`. If it's necessary, use `_` or `_severity` as an argument name to indicate that it won't be used.<br>Unused block argument - `progname`. If it's necessary, use `_` or `_progname` as an argument name to indicate that it won't be used.